mobile-api/docker-compose.yml

```
@@ -0,0 +1,21 @@
+version: '3'
+services:
+  mongodb:
+    image: mongo:4.0
+    restart: always
+    volumes:
+      - 'mongodb:/data/db'
+  api:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    depends_on:
+      - mongodb
+    environment:
+      - MONGODB_URL
```

```suggestion
      - MONGODB_URL=mongodb://mongodb:27017/mobile-api
```
I hadn't properly understood how this was to be used, until @raisedadead's comment
> The compose file will not have much use other than local development/testing.

Since this is only supposed to be run in dev/CI your original approach seems good.  Sorry for the distraction!